examples/compose-replication.yml

```diff
@@ -1,34 +1,67 @@
 version: "3"
+x-common-variables: &common-variables
+  MARIADB_ROOT_PASSWORD: ${MARIADB_ROOT_PASSWORD}
+  MYSQL_INITDB_SKIP_TZINFO: ${MYSQL_INITDB_SKIP_TZINFO}
+  MARIADB_REPLICATION_USER: ${MARIADB_REPLICATION_USER}
+  MARIADB_REPLICATION_PASSWORD: ${MARIADB_REPLICATION_PASSWORD}
+
+x-common-attributes: &common-attributes
+  image: mariadb:lts
+  env_file:
+    - .env
+  environment: *common-variables
+  healthcheck:
+    test: ["CMD", "healthcheck.sh", "--connect", "--innodb_initialized"]
+    interval: 10s
+    timeout: 5s
+    retries: 3
+  logging:
+    driver: journald
+  networks:
```

Comment:
Isn't a dedicated network created for the compose file? Is there something gained by this?